app/navbar.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import Image from "next/image";
+import WalletButton from "@/app/walletButton";
+
+export default function Navbar() {
+  return (
+    <nav className="flex w-auto h-[86px] px-5 pl-10 py-5 
```

Comment:
This applies to any usage of tailwinds:

**Write reusable tailwind components. Do not write inline css**.

Reply:
Not sure I fully understand, can you please provide an example of how it would look like?

Reply:
IMO if the designs do not follow the sizing scale, we should pick a tailwinds size closest to the desired value. For example, h-20 or h-24 instead of h-[85px].